espei/run_espei.py

```diff
@@ -135,25 +135,31 @@ def main():
             # code recommended by emcee: if not master, wait for instructions then exit
             client = MPIPool()
             if not client.is_master():
-                logging.debug(
+                logging.warning(
```

Comment:
Make sure your commits are rebased on top of the HEAD of PhasesResearchLab/ESPEI so that my previous commits are not undone.